packages/react/src/components/AccountSettings/ChangePassword/types.ts

```diff
@@ -1,5 +1,18 @@
 import { InputEventType, ValidatorOptions } from '@aws-amplify/ui';
-import { FormValues } from '../types';
+import {
+  AccountSettingsError,
+  AccountSettingsPasswordField,
+  AccountSettingsSubmitButton,
```

Comment:
Should we remove the "AccountSettings" prefix for these and append "Component" to align with other connected component override types?

Reply:
SGTM

Reply:
Done in [8bd8026](https://github.com/aws-amplify/amplify-ui/pull/3075/commits/8bd802604bfa49d2ff259b7564eb8dc61e7d7ca9)

Reply:
Typos corrected in e59c84b013b181c8ac3b7214652226a2ef07ddd6